lib/autocomplete-manager.coffee

```diff
@@ -395,3 +399,24 @@ class AutocompleteManager
     @subscriptions = null
     @suggestionList = null
     @providerManager = null
+
+  clearManualActivationStrictPrefixes: ->
+    @manualActivationStrictPrefixes = null
+
+  addManualActivationStrictPrefix: (provider, prefix) ->
+    return if @manualActivationStrictPrefixes?.has(provider) or not prefix?
+    @manualActivationStrictPrefixes ?= new WeakMap
+    @manualActivationStrictPrefixes.set(provider, prefix.toLowerCase())
+
+  filterForManualActivationStrictPrefix: (suggestions) ->
+    return suggestions unless @manualActivationStrictPrefixes?
+
+    results = []
+    for suggestion in suggestions
+      lowercaseText = (suggestion.snippet ? suggestion.text).toLowerCase()
+      if lowercaseText[0] is suggestion.replacementPrefix.toLowerCase()[0]
```

Comment:
@nathansobo I'm just doing a first char match here to qualify suggestions for strict matching. Otherwise all fuzzy matched suggestions would pass through this without filtering. e.g.

```
{text: 'sabomething', replacementPrefix: 'some'} 
```